robot_smach_states/examples/example_hear_grasp.py

```diff
@@ -13,13 +13,13 @@ def __init__(self, robot):
         self.robot = robot
 
     def execute(self, userdata):
-        result = self.robot.ears.recognize(spec='Bring me a <drink>',
-                                           choices={'drink': ['coke', 'fanta']},
-                                           time_out=rospy.Duration(60))
-        choices = result.choices['drink'] if result else None
-        if choices:
-            rospy.loginfo('I heared:' + repr(choices))
-        userdata.heared = choices
+        answer = self.robot.hmi.query(description='can i have a coke?',
```

Comment:
Now we can only hear coke....

Reply:
This is an example for query I suppose, so we are not hearing anything from the Robot.

Reply:
Please restore old coke/fanta option